miio/gateway/devices/subdevice.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         if model_info is None:
             model_info = {}
         self._model_info = model_info
+        self._battery_powered = model_info.get("battery", True)
```

Comment:
```suggestion
        self._battery_powered = model_info.get("battery_powered", True)
```

What about calling the model info property `battery_powered` too? Just for consistency.

Reply:
sure I will change it

Reply:
done